afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/NodeInfo.java

```diff
@@ -54,6 +54,10 @@ public String getName() {
         return name;
     }
 
+    public void setName(String name) {
+        this.name = Objects.requireNonNull(name);
```

Comment:
Empty names should not be allowed: maybe we have to check for that too. Introduce a new private and **static** method called checkName to ensure that name is not null and not empty and use it in this setter and the constructor.
```
public void setName(String name) {
    this.name = checkName(name);
}
```

Reply:
it should return what if the name is empty ?, an exception?